src/connector/tatum.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios'
+import { plainToClass } from 'class-transformer'
+import { ClassType } from 'class-transformer/ClassTransformer'
+import { validateOrReject } from 'class-validator'
+import { TATUM_API_URL } from '../constants'
+
+const baseUrl = () => `${process.env.TATUM_API_URL || TATUM_API_URL}`
```

Comment:
I tried it, but unfortunately when env variable is out of scope the value is not available. I will try to double check it again but I think it will be same result.